app/Model/Attribute.php

```diff
@@ -660,6 +660,11 @@ private function __alterAttributeCount($event_id, $increment = true)
 
     public function afterSave($created, $options = array())
     {
+       //STR
```

Comment:
![image](https://user-images.githubusercontent.com/3668672/74220157-1dbaf180-4caf-11ea-9df2-689b33a1be32.png)

Reply:
damn, this was a test I left :( let me remove that

Reply:
Thanks :)